tools/pyclient/dev/dev.py

```diff
@@ -0,0 +1,85 @@
+#///////////////////////////////////////////////////////////////////////////////
+# FILE: dev.py
+# AUTHOR: David Ruvolo
+# CREATED: 2023-05-22
+# MODIFIED: 2023-07-21
+# PURPOSE: development script for initial testing of the py-client
+# STATUS: ongoing
+# PACKAGES: pandas
+# COMMENTS: Designed to interact with the schema "pet store".
+#///////////////////////////////////////////////////////////////////////////////
+
+import pandas as pd
+
+from tools.pyclient.src.molgenis_emx2_pyclient import Client
+
+# connect and sign in
+db = Client('https://emx2.dev.molgenis.org/')
+db.sign_in('', '')
+
+# check status
+db.status()
+
+# get data
+data = db.get(schema='', table='') # run without specifying target
+data = db.get(schema = 'pet store', table='') # run without specifying table
+data = db.get(schema = 'pet store', table='Pet') # get Pets
+data
+
+data = db.get(schema = 'pet store', table='Pet', as_df=True) # get Pets
+
+#///////////////////////////////////////////////////////////////////////////////
+
+# ~ 1 ~
+# Check Import Methods
+
+# ~ 1a ~
+# Check import via the `data` parameters
+# Add new record to the pet store with new tags
+
+newTags = [
+    {'name': 'brown', 'parent': 'colors'},
+    {'name': 'canis', 'parent': 'species'},
+]
+
+newPets = [{
+    'name': 'Woofie',
+    'category': 'dog',
+    'status': 'available',
+    'weight': 6.8,
+    'tags': 'brown,canis'
+}]
+
+# import new data
+db.add(schema='pet store', table='Tag', data=newTags)

```

Comment:
in the Java  version I had a preference to do this on schema level so I don't need to provide schema all the time.
In javascript we now give option to set schema once and then it assumes that schema.

Reply:
We've discussed this as well and the client will likely follow the javascript version. We wanted to have some flexibility when working with multiple schemas (i.e., passing data between staging areas).